core/arch/arch.c

```diff
@@ -3355,45 +3355,59 @@ dr_mcontext_to_priv_mcontext(priv_mcontext_t *dst, dr_mcontext_t *src)
      * if we append to dr_mcontext_t in the future we'll need
      * to check src->size here.
```

Comment:
The 2nd sentence of this comment is stale (already implemented): can be removed.

Reply:
Removed